mandatory/4-tax.js

```diff
@@ -17,8 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(num) {
+  var taxAndCurrency = calculateSalesTax(num).toFixed(2)
 
+return `£${taxAndCurrency}`
```

Comment:
Great submission again Cuneyt, the only thing I'd remind you is that we need to keep indentation consistent 👍